### PR TITLE
docs(ecosystem): initial categorization of plugins

### PR DESCRIPTION
**Pull request by JonHPC**
Closes https://github.com/fastify/fastify/issues/3618
Sorted Plugins into categories based on https://fastify.github.io/pacchetto/

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by Eomm — COMMENTED** [review]
This impact the website build, we need to update it before merging

I would like a tag column instead of a new paragraph. What if a plugin does multiple things?

**Comment by JonHPC** [comment]
Oh true, I suppose a tag column would make more sense for multiple categories. I'll try to re-work it, thanks!

**Comment by JonHPC** [comment]
Screenshot 2023-07-29 at 4 11 01 PM
How about arranging it like this? A table with 3 columns, one for the plugin, one for description, and one for tags.

**Review by Eomm — COMMENTED** [review]
last feedback: tag lowercase

then LGTM!

**Review by jsumners — CHANGES_REQUESTED** [review]
Does https://github.com/fastify/fastify/blob/298adbc3cf32b72998736eb503ab61d01c191562/.github/workflows/lint-ecosystem-order.yml still work?

**Comment by JonHPC** [comment]
> Does https://github.com/fastify/fastify/blob/298adbc3cf32b72998736eb503ab61d01c191562/.github/workflows/lint-ecosystem-order.yml still work?

Hmm, I'm not too sure

**Comment by metcoder95** [comment]
> > Does [`298adbc`/.github/workflows/lint-ecosystem-order.yml](https://github.com/fastify/fastify/blob/298adbc3cf32b72998736eb503ab61d01c191562/.github/workflows/lint-ecosystem-order.yml) still work?
> 
> Hmm, I'm not too sure

It is possible to make it work or with the new format it's not? 🙁

**Comment by Eomm** [comment]
Any update on this?

What is missing:

- [ ] update the lint check https://github.com/fastify/fastify/blob/298adbc3cf32b72998736eb503ab61d01c191562/.github/workflows/lint-ecosystem-order.yml
- [ ] propagate the change to the community plugin
- [ ] for me: update the website before merging this PR

File: docs/Guides/Ecosystem.md [hunk 1]
```diff
@@ -7,134 +7,62 @@ plugins maintained by the community are listed in the [Community](#community)
 section.
 
 #### [Core](#core)
+| Plugin | Description | Tags |
+| - | - | - |
+| [`@fastify/accepts`](https://github.com/fastify/fastify-accepts) |  to have[accepts](https://www.npmjs.com/package/accepts) in your request object. | `http` |
```
Comment (codershiba) [inline docs/Guides/Ecosystem.md:12]:
```suggestion
| [`@fastify/accepts`](https://github.com/fastify/fastify-accepts) |  To have [accepts](https://www.npmjs.com/package/accepts) in your request object. | `http` |
```

File: docs/Guides/Ecosystem.md [hunk 2]
```diff
@@ -7,134 +7,62 @@ plugins maintained by the community are listed in the [Community](#community)
...
+| [`@fastify/basic-auth`](https://github.com/fastify/fastify-basic-auth) | Basic auth plugin for Fastify. | `authentication`   |
+| [`@fastify/bearer-auth`](https://github.com/fastify/fastify-bearer-auth) | Bearer auth plugin for Fastify. | `authentication`   |
+| [`@fastify/caching`](https://github.com/fastify/fastify-caching) | General server-side cache and ETag support. | `caching`   |
+| [`@fastify/circuit-breaker`](https://github.com/fastify/fastify-circuit-breaker) | A low overhead circuit breaker for your routes. | `routing`    |
+| [`@fastify/compress`](https://github.com/fastify/fastify-compress) | Fastify compression utils. | `http`   |
+| [`@fastify/cookie`](https://github.com/fastify/fastify-cookie) | Parse and set cookie headers. | `http`   |
+| [`@fastify/cors`](https://github.com/fastify/fastify-cors) | Enables the use of CORS in a Fastify application. | `http`   |
+| [`@fastify/csrf-protection`](https://github.com/fastify/csrf-protection) | A plugin for adding[CSRF](https://en.wikipedia.org/wiki/Cross-site_request_forgery) protection to Fastify. | `http`   |
```
Comment (codershiba) [inline docs/Guides/Ecosystem.md:26]:
```suggestion
| [`@fastify/csrf-protection`](https://github.com/fastify/csrf-protection) | A plugin for adding [CSRF](https://en.wikipedia.org/wiki/Cross-site_request_forgery) protection to Fastify. | `http`   |
```

File: docs/Guides/Ecosystem.md [hunk 3]
```diff
@@ -7,134 +7,62 @@ plugins maintained by the community are listed in the [Community](#community)
...
+| [`@fastify/jwt`](https://github.com/fastify/fastify-jwt) | JWT utils for Fastify, internally uses [fast-jwt](https://github.com/nearform/fast-jwt). | `authentication`   |
+| [`@fastify/leveldb`](https://github.com/fastify/fastify-leveldb) | Plugin to share a common LevelDB connection across Fastify. | `databases`   |
+| [`@fastify/middie`](https://github.com/fastify/middie) | Middleware engine for Fastify. | `compatability`   |
+| [`@fastify/mongodb`](https://github.com/fastify/fastify-mongodb) | Fastify MongoDB connection plugin, with which you can share the same MongoDB connection pool across every part of your server. | `databases`    |
+| [`@fastify/multipart`](https://github.com/fastify/fastify-multipart) | Multipart support for Fastify. | `http`   |
+| [`@fastify/nextjs`](https://github.com/fastify/fastify-nextjs) | React server-side rendering support for Fastify with[Next](https://github.com/zeit/next.js/). |    |
+| [`@fastify/oauth2`](https://github.com/fastify/fastify-oauth2) | Wrap around[`simple-oauth2`](https://github.com/lelylan/simple-oauth2).    | `authentication`   |
```
Comment (codershiba) [inline docs/Guides/Ecosystem.md:43]:
```suggestion
| [`@fastify/oauth2`](https://github.com/fastify/fastify-oauth2) | Wrapper around [`simple-oauth2`](https://github.com/lelylan/simple-oauth2).    | `authentication`   |
```

File: docs/Guides/Ecosystem.md [hunk 4]
```diff
@@ -7,134 +7,62 @@ plugins maintained by the community are listed in the [Community](#community)
...
+| [`@fastify/reply-from`](https://github.com/fastify/fastify-reply-from) | Plugin to forward the current HTTP request to another server. | `networking`   |
+| [`@fastify/request-context`](https://github.com/fastify/fastify-request-context) | Request-scoped storage, based on[AsyncLocalStorage](https://nodejs.org/api/async_hooks.html#async_hooks_class_asynclocalstorage)(with fallback to [cls-hooked](https://github.com/Jeff-Lewis/cls-hooked)), providing functionality similar to thread-local storages. |    |
+| [`@fastify/response-validation`](https://github.com/fastify/fastify-response-validation) | A simple plugin that enables response validation for Fastify. | `validation`   |
+| [`@fastify/routes`](https://github.com/fastify/fastify-routes) | Plugin that provides a `Map` of routes. | `routing`    |
+| [`@fastify/schedule`](https://github.com/fastify/fastify-schedule) | Plugin for scheduling periodic jobs, based on[toad-scheduler](https://github.com/kibertoad/toad-scheduler). | `scheduling`   |
```
Comment (codershiba) [inline docs/Guides/Ecosystem.md:52]:
```suggestion
| [`@fastify/schedule`](https://github.com/fastify/fastify-schedule) | Plugin for scheduling periodic jobs, based on [toad-scheduler](https://github.com/kibertoad/toad-scheduler). | `scheduling`   |
```

**Comment by codershiba** [comment]
Why was this closed

**Comment by Uzlopak** [comment]
Uff. it was opened 9 months ago :/

**Pull request closed without merging**